### PR TITLE
Connect to API

### PR DESCRIPTION
**Pull request by gak92**
In this pull request: 

- Used `fetch` for API calls
- Used `thunk`
- Display book list by fetching data from API
- Update Action Creator of Add and Remove so that returns the function

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -38,20 +24,64 @@ const booksReducer = (state = initialState, action) => {
...
+    return {
+      id: key,
+      title,
+      author,
+      category,
+    };
+  });
+  if (books) {
+    dispatch({
+      type: DISPLAY_BOOKS,
```
Comment (BertrandConxy) [inline src/redux/books/books.js]: - [ ] Good job so far 👏 As it is among project requirements, we are expected to use this new function provided by the redux toolkit package called [`createAsyncThunk`](https://redux-toolkit.js.org/api/createAsyncThunk) which makes it easier to create Action creators in our React-redux applications. 
image

**Please refactor your action creators in order to use this new way of using the createAsyncThunk** 😉

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @gak92 ✋ , 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 


## Highlights 💪 
- No linter errors
- correct flow is used
- good PR
- project works fine


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
Hi @gak92, 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by gak92** [comment]
@t-yanick 
Thanks for the feedback

**Pull request merged**